### ISSUE TITLE
Quick run dont work when use custom executor

### ISSUE THREAD

**Issue by xiejay97** [nrwl/nx-console#1125]
When use custom executor:
I click **1**, there no response.
I click **2**, it's ok.
![image](https://user-images.githubusercontent.com/64340763/130417115-fef56c40-7d87-4e21-a28b-b6c7b9fb0404.png)

https://user-images.githubusercontent.com/64340763/130417661-b1c2235f-d214-4fc8-8698-15c782aefd20.mp4

**Comment by Cammisuli** [comment]
hmmmm.. I use this feature every day. And the only thing I see that could be different is yours being on Windows. I really got to get through all these Windows issues ☹️

**Comment by xiejay97** [comment]
> hmmmm.. I use this feature every day. And the only thing I see that could be different is yours being on Windows. I really got to get through all these Windows issues ☹️

@Cammisuli You are right,  I use Windows host and install vscode on it. A bit more, I use vscode by wsl2 whose system version is Ubuntu 20.04.

**Comment by Cammisuli** [comment]
There's also a fact I missed - a _local_ custom executor. So I'll need to check that too.

**Comment by Cammisuli** [comment]
@xiejay97 can you try this again with the latest version? I made some changes to how tasks are executed and will now use your default configured terminal in VS Code. So if you have wsl set as your default, all these tasks should run in wsl. 

In the meantime, I'm going to close this because this should've fixed it. If not, reply here and I'll reopen.

**Comment by xiejay97** [comment]
> @xiejay97 can you try this again with the latest version? I made some changes to how tasks are executed and will now use your default configured terminal in VS Code. So if you have wsl set as your default, all these tasks should run in wsl.
> 
> In the meantime, I'm going to close this because this should've fixed it. If not, reply here and I'll reopen.

Its OK for me now, thanks for your job ❤️ @Cammisuli

**Comment by zanettin** [comment]
sorry @Cammisuli to chime in here as well 🙈  have the same issue as mentioned by @xiejay97. clicking on the link within the `project.json` does nothing. click on the `run` cmd does also nothing. only clicking on the `projects` actions does trigger my custom action. any ideas what could be the cause of it? thanks a lot in advance 🙏 

https://user-images.githubusercontent.com/3241476/143937339-723f88cc-4da7-4e10-9162-91fe21bd9742.mp4

### computer specs
mac book pro 2021
osx monterey
vs code insiders `1.63.0-insider``
nx console `17.12.4`
iterm2 terminal with oh-my-zsh installed


if you need any further info just let me know. thanks again
